custom_components/places/update_sensor.py

```
@@ -377,18 +473,22 @@ async def update_coordinates(self) -> None:
                 tracker_id,
             )
             return
         if tracker_snapshot.status == TrackerStatus.OK:
             if tracker_snapshot.latitude is not None:
-                self.sensor.set_attr(ATTR_LATITUDE, tracker_snapshot.latitude)
+                self.coordinator.set_attr(ATTR_LATITUDE, tracker_snapshot.latitude)
             if tracker_snapshot.longitude is not None:
-                self.sensor.set_attr(ATTR_LONGITUDE, tracker_snapshot.longitude)
+                self.coordinator.set_attr(ATTR_LONGITUDE, tracker_snapshot.longitude)
+
+    async def determine_update_criteria(self, force: bool = False) -> UpdateStatus:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)